connectors/php/BaseFilemanager.php

```diff
@@ -236,7 +236,7 @@ public function handleRequest()
                         break;
 
                     case 'getfolder':
-                        if($this->getvar('path') && $this->getvard('skip', true, 0)) {
+                        if($this->getvar('path') && $this->getvard('skip', true, 0) && $this->getvard('load', true, 0)) {
```

Comment:
Take a look for new function `getvard` which don't throw error if variable wasn't set. 3rd argument is default value which will be used if name not defined in $_GET.